core/src/main/java/io/kestra/core/serializers/YamlFlowParser.java

```diff
@@ -97,5 +101,41 @@ private Flow readFile(File file) {
             );
         }
     }
+
+    private Flow readString(String input) {
+        try {
+            return mapper.readValue(input, Flow.class);
+        } catch (JsonProcessingException e) {
+            if (e.getCause() instanceof ConstraintViolationException) {
+                throw (ContainedValueException) e.getCause();
+            } else {
+                throw new ConstraintViolationException(
```

Comment:
Here you loose the cause of the exception, I'm not a big fan as you loose usefull debugging context.

Reply:
ConstraintViolationException are capture on the webserver and will be hidden since it's a user generated exception, we never display full stacktrace